cli/failover/set.go

```diff
@@ -0,0 +1,79 @@
+package failover
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/apex/log"
+	"github.com/tarantool/cartridge-cli/cli/context"
+)
+
+func Set(ctx *context.Ctx) error {
+	if (ctx.Failover.Mode == "eventual" || ctx.Failover.Mode == "disabled") && ctx.Failover.ProviderParamsJSON != "" {
```

Comment:
There is a `timeout` parameter for `eventual` mode. Seems that it supports parameters as well as stateful.

Reply:
``ProviderParamsJSON`` is ``stateboard_params`` or ``etcd2_params``, not parameters like ``failover_timeout``, ``fencing_enabled``, etc - they are saved in ``FailoverParamsJSON`` variable (If you talking about this params)